socketmode/socket_mode_managed_conn.go

```diff
@@ -162,6 +162,7 @@ func (smc *Client) connect(ctx context.Context, connectionCount int, additionalP
 		errInvalidAuth      = "invalid_auth"
 		errInactiveAccount  = "account_inactive"
 		errMissingAuthToken = "not_authed"
+		errInvalidToken     = "invalid_token"
```

Comment:
invalid_token? token_revoked? 🤔

Reply:
Does Slack send us to different errors for SocketMode and RTM??

Reply:
apologies, I messed up, it's the same error string. Fixing it